src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -98,6 +102,7 @@ class DecisionTree :
    * @param numClasses Number of classes in the dataset.
    * @param weights The weight list of given label.
    * @param minimumLeafSize Minimum number of points in each leaf node.
+   * @param minimumGainSplit Minimum Gain for the node to split.
```

Comment:
This is super picky (sorry!) but you can change `Gain` to `gain` here---there's no need to capitalize. :)